prisma/devSetup.ts

```diff
@@ -0,0 +1,211 @@
+/* eslint-disable func-names */
+/* eslint-disable no-console */
+import { PrismaClient } from '@prisma/client';
+import loadCSVFromFile from './loadCsv';
+
+const seedHealthcareProfessionals = async function (prisma: PrismaClient, verbose = false) {
+  const firstEn = 0;
+  const middleEn = 1;
+  const lastEn = 2;
+  const firstJa = 3;
+  const middleJa = 4;
+  const lastJa = 5;
+  const degreeCol = 6;
+  const specialtyCol = 7;
+  const spokenLangCol = 8;
+
+  const devData:string[][] = loadCSVFromFile('./prisma/seedData/devHealthcareProfessionals.csv');
+
+  devData.forEach(async (row: string[], index: number) => {
+    // generate some unique ids using even/odd strategy
+    const enID = 2 * index;
+    const jaID = 2 * index + 1;
+
+    // Make a nested write
+    const newPersonName = await prisma.personName.upsert({
+      where: { id: index },
+      update: {},
+      create: {
+        id: index,
+        names: {
+          create: [
+            {
+              id: enID,
+              locale: 'en',
+              firstName: row[firstEn],
+              middleName: row[middleEn],
+              lastName: row[lastEn],
+            },
+            {
+              id: jaID,
+              locale: 'ja',
+              firstName: row[firstJa],
+              middleName: row[middleJa],
+              lastName: row[lastJa],
+            },
+          ],
+
+        },
+      },
+    });
+
+    if (verbose) {
+      console.log(`Inserted ${newPersonName.id} into PersonName`);
+    }
+
+    // create a healthcare professional
+    const newHealthPro = await prisma.healthcareProfessional.upsert({
+      where: { id: index },
+      update: {},
+      create: {
+        id: index,
+        personNameId: newPersonName.id,
+        isPublished: true,
+      },
+    });
+
+    // Link spoken languages
+    const spokenLangList = row[spokenLangCol].split(',');
+    spokenLangList.forEach(async (lang) => {
+      const dbSpokenLang = await prisma.spokenLanguage.findFirst(
+        {
+          where: {
+            iso639_3: lang,
+          },
+        },
+      );
+      if (dbSpokenLang) {
+        // TODO change to upsert
```

Comment:
It would be better if I could make this an upsert so that running the seed file multiple times does not result in duplicate data